Energy Efficiency EACs Using Pre and Post Measurements.md

```diff
@@ -0,0 +1,516 @@
+Name: Energy Efficiency EACs using pre and post measurements
+
+Developer: **C3**
+
+Revision Date: **2024-06-20**
+
+**Glossary**
+
+Asset: An energy device or energy system within a measurement boundary acted upon by an intervention that results in a change in energy consumption relative to a baseline. 
+
+Site: A uniquely identifiable geographical location of an Asset
+
+Pre: Denotes the time prior to the intervention that will change energy consumption
+
+Post: Denotes the time after the intervention that will change energy consumption.
+
+**Summary**
+
+The purpose of this methodology is to estimate the energy and related carbon impact of energy efficiency projects in commercial buildings. Specifically, the methodology looks to measure the number of hourly useful EACs, avoided emissions, and the generated grid emissions from changing electricity consumption.
+
+The applicable interventions will be listed in the Measure Upgrades section.
+
+**Measure Upgrades**
+
+1. LED Lighting and / or Lighting Controls (including dimming)
+
+**Project Stakeholders**
+
+Provide contact information for relevant stakeholders which may include Asset Owner.
```

Comment:
Hypothetically this EAC might be a tradable asset, moving beyond the project team. Does it make sense to have the contact information in here? What's the benefit of the contact info being here?